controllers/articles.js

```diff
@@ -19,29 +19,37 @@ const getAverageRating = (data) => {
   return Number(average.toFixed(2));
 };
 
-
 const articles = {
-
   /*
-   * Crteating an article.
+   * Creating an article.
    *
    * For directly publishing an article, status flag has to be passed
    * into the request body object otherwise the article will be in drafted.
    */
   createArticle: async (req, res) => {
     try {
       const {
-        title, body, status, tagList,
+        title, body, status, tagList
       } = req.body;
       const { id: author } = req.user;
-      const flag = status === undefined ? 'Draft' : status;
+      const flag = status === undefined ? 'draft' : status;
```

Comment:
I see here you are changing the status and published from Capitalized word to small case, for avoiding the conflict that may occur when someone sends `DRAFT` from the body and compare it to `draft` here it will return that they are not equal. So to avoid that you can `toLowerCase()` when storing status to DB and when comparing them. (Better if you check on all endpoints where matters and fix that if you chose to convert it into lowercase)